features/org.wso2.carbon.identity.oauth.server.feature/resources/oauth_response.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+
+<!--
+~    Copyright (c) 2020, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
+~
+~    This software is the property of WSO2 Inc. and its suppliers, if any.
+~    Dissemination of any information or reproduction of any material contained
+~    herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+~    You may not alter or remove any copyright or other notice from copies of this content."
+-->
+
+<html>
+
+<head>
+    <title>Redirecting</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css">
+
+    <style>
+        :root {
+            --primary-color: #FF5500;
+        }
+
+        .ui.primary.button {
+            background-color: var(--primary-color) !important;
+        }
+
+        div.redirect-loader {
+            border: 1px solid #CDCDCD;
+            padding: 20px;
+            border-radius: 6px;
+            text-align: center;
+            font-size: 1.2em;
+        }
+
+        .center {
+            margin: 0;
+            position: relative;
+            left: 50%;
+            -ms-transform: translate(-50%, -50%);
+            transform: translate(-50%, -50%);
+        }
+
+        .padding-top-20 {
+            padding-top: 20rem;
+        }
+
+        .padding-top-2 {
+            padding-top: 4rem;
+        }
+
+        .padding-bottom-1 {
+            padding-bottom: 1rem;
+        }
+
+        .text-align-center {
+            text-align: center;
+        }
+
+        .loader-redirect {
+            font-size: 17px;
+            color: #787778;
+        }
+    </style>
+</head>
+
+<body onload="javascript:document.forms[0].submit()">
+
+    <div class="padding-top-20">
+        <div class="ui centered grid">
+            <div class="twelve wide mobile eight wide tablet six wide computer column">
+                <div class="redirect-loader">
+                    <i class="fa fa-circle-o-notch fa-spin loader-redirect"></i>
+                    Please wait while you're being redirected...
```

Comment:
We are doing an automatic form post here. So "you're being redirected" might give a wrong impression that we are doing a redirect.

@ayshsandu @janakamarasena WDYT?

Reply:
What about "Please wait while we are processing your request" ?

Reply:
Need to change the next message also accordingly. `Click the submit button if automatic redirection failed.` -> `Click the submit button if automatic submission failed.` ?

Reply:
> What about "Please wait while we are processing your request" ?

How about
"Please wait while we complete your request"

Reply:
Fixed with fc4b8cf

Reply:
We changed the phrases to the following after a discussion with @ayshsandu 

Main message
`Please wait while we take you to the application.`

The description for action button 
`Click continue if you have been waiting for too long.`

@mefarazath WDYT of that approach?

Reply:
+1